src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Crypto.cs

```diff
@@ -164,5 +167,52 @@ internal static byte[] GetDynamicBuffer<THandle>(NegativeSizeReadMethod<THandle>
 
             return bytes;
         }
+
+        [LibraryImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_GetMemoryUse")]
+        private static partial int GetMemoryUse(ref long memoryUse, ref long allocationCount);
```

Comment:
This method returns `int`, but you never check it.

Since the only callers are guaranteed to pass non-null, you can change the target function to be void-returning and just `assert` that each of the pointers is not null.